src/plugins/dashboard/public/application/utils/use/use_saved_dashboard_instance.ts

```diff
@@ -40,85 +40,103 @@ export const useSavedDashboardInstance = (
       http: { basePath },
       notifications,
       toastNotifications,
+      data: {
+        query: {
+          timefilter: { timefilter },
+        },
+      },
     } = services;
 
+    const handleErrorFromSavedDashboard = (error: any) => {
+      // Preserve BWC of v5.3.0 links for new, unsaved dashboards.
+      // See https://github.com/elastic/kibana/issues/10951 for more context.
```

Comment:
Did we look at this? is it still relevant?

Reply:
It's still supported and verified within the tests. And the context helps. If we wanted we can make a request for docs that state replacing the query string param in the url with an older version.